### PR TITLE
yasser addition

### PR DESCRIPTION
**Pull request by yasserh2015**

### CODE REVIEW

File: GitTest/SecondViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
```
Comment (sam961) [inline GitTest/SecondViewController.swift:15]: remove comments here

Comment (yasserh2015) [reply GitTest/SecondViewController.swift:15]: will do

Comment (sam961) [reply GitTest/SecondViewController.swift:15]: i will add a review for this

File: GitTest/SecondViewController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+//  Created by Yasser Hamdan on 14/01/2022.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (sam961) [inline GitTest/SecondViewController.swift:14]: remove spaces are done

**Review by sam961 — APPROVED** [review]

**Pull request merged**